src/ReleaseHistory.md

```diff
@@ -3,6 +3,7 @@
 ## **v2.4.9** [Sdk](https://www.nuget.org/packages/Sarif.Sdk/2.4.9) | [Driver](https://www.nuget.org/packages/Sarif.Driver/2.4.9) | [Converters](https://www.nuget.org/packages/Sarif.Converters/2.4.9) | [Multitool](https://www.nuget.org/packages/Sarif.Multitool/2.4.9) | [Multitool Library](https://www.nuget.org/packages/Sarif.Multitool.Library/2.4.9)
 * FEATURE: Report inner exception details if available. [#2357](https://github.com/microsoft/sarif-sdk/pull/2357)
 * FEATURE: Add support for git blame. [#2358](https://github.com/microsoft/sarif-sdk/pull/2358)
+* FEATURE: Add Clang-Tidy converter. [#2367](https://github.com/microsoft/sarif-sdk/pull/2367)
```

Comment:
>FEATURE: Add Clang-Tidy converter. [](http://example.com/codeflow?start=2&length=34)

move this to line 2.
This change wasn't released in 2.4.9.

Reply:
got it, fixed